go/kbfs/libfs/fs.go

```diff
@@ -691,7 +693,7 @@ func (fs *FS) Rename(oldpath, newpath string) (err error) {
 		err = translateErr(err)
 	}()
 
-	if err := fs.requireNonEmpty(); err != nil {
+	if err := fs.checkEmpty(false); err != nil {
```

Comment:
Why `false` here? Shouldn't we error if the fs is empty?

Reply:
We still error, but get `os.ErrNotExist`, which is consistent with when you try to `mv non-existing-file some-other-name` in shell. But I can see this returning the other error too. Not really sure how to decide.